gradle/ext/login-web-ext/build.gradle

```diff
@@ -0,0 +1,11 @@
+apply plugin: 'com.liferay.osgi.ext.plugin'
+
+dependencies {
+	compileOnly group: "com.liferay.portal", name: "com.liferay.portal.impl", version: "default"
```

Comment:
We can't use `default` version.  I don't even know what that does.

Reply:
changed to 3.0.0